README.md

```diff
@@ -1,13 +1,6 @@
 # dw-ripple
 
-Ripple element created as child-class of [`mwc-ripple`]
-(https://github.com/material-components/material-components-web-components/tree/master/packages/ripple)
-
-## What are the changes in `mwc-ripple`?
-- To show ripple in secondary color, they have used property named `accent`. Which is confusing in terms of standard 
-theming variables.
-- They render ripple always in black color by default. So, it doesn't work on dark theme. We changed default color to 
-`--mdc-theme-on-surface`
+Shows a ripple effect based ripple color and opacity.
```

Comment:
Change this to "Shows ripple effect on the click of the parent element. By default ripple is shown in the `on-surface` color. But, you have options to show it either in the primary, secondary or even any other color."